csrankings-r.csv

```diff
@@ -688,7 +688,7 @@ Richard Peng,University of Waterloo,http://www.cc.gatech.edu/~rpeng,loh7ivYAAAAJ
 Richard Ryan Williams,Massachusetts Institute of Technology,http://web.stanford.edu/~rrwill,Wp38QQYAAAAJ
 Richard S. Bird,University of Oxford,http://www.cs.ox.ac.uk/richard.bird,1vrc8EUAAAAJ
 Richard S. Sutton,University of Alberta,https://webdocs.cs.ualberta.ca/~sutton,hNTyptAAAAAJ
-Richard S. Zemel,Columbia University,http://www.cs.toronto.edu/~zemel,iBeDoRAAAAAJ
+Richard S. Zemel,University of Toronto,http://www.cs.toronto.edu/~zemel,iBeDoRAAAAAJ

```

Comment:
Web page says "NOTE: I am on-leave from the University of Toronto in 2021-2022, and am not taking on new students." Please clarify.